app/rdf/query-hierarchies.ts

```diff
@@ -127,25 +133,25 @@ export const queryHierarchy = async (
     cache,
   });
 
-  const allHierarchies = await Promise.all(
-    hierarchies?.map(async (h) => ({
-      // @ts-ignore
-      nodes: (await getHierarchy(h).execute(sparqlClientStream, rdf)) || [],
-      hierarchyName: getName(h.out(ns.cubeMeta.nextInHierarchy), locale),
-    }))
+  const hierarchies = await Promise.all(
+    hierarchyPointers.map(async (pointer) => {
+      return {
+        // @ts-ignore
+        nodes: await getHierarchy(pointer).execute(sparqlClientStream, rdf),
+        hierarchyName: getName(pointer, locale),
+      };
```

Comment:
Why the @ts-ignore ? Also i think you can save the `return` and just keep it inline

Reply:
Here the types are quite messy, see e.g. the image below – I tried to fix them, but failed 😅 But it should still work though, as we had this comment previously 👍 

<img width="1041" alt="Screenshot 2023-11-10 at 11 55 24" src="https://github.com/visualize-admin/visualization-tool/assets/52032047/a615c642-f79c-4d0e-bc80-5e3f30a5bac7">